### ISSUE TITLE
Polyfill utilities throws "TypeError: Cannot read property 'toLowerCase' of undefined"

### ISSUE THREAD

**Issue by dwiyatci** [hotforfeature/origami#85]
I've got the following error when running `./node_modules/.bin/origami polyfill`:

```cmd
Origami: TypeError: Cannot read property 'toLowerCase' of undefined
    at isEs5 (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/@codebakery/origami/bin/lib/polyfill/util.js:80:44)
    at Object.isAngularJsonEs5 (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/@codebakery/origami/bin/lib/polyfill/util.js:65:16)
    at /Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/@codebakery/origami/bin/lib/polyfill/add-polyfill-assets.js:25:60
    at Array.forEach (<anonymous>)
    at _loop_1 (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/@codebakery/origami/bin/lib/polyfill/add-polyfill-assets.js:22:39)
    at Object.<anonymous> (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/@codebakery/origami/bin/lib/polyfill/add-polyfill-assets.js:33:29)
    at step (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/tslib/tslib.js:133:27)
    at Object.next (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/tslib/tslib.js:114:57)
    at fulfilled (/Users/glenn/github/c8y-angular-polymer-boilerplate/node_modules/tslib/tslib.js:104:62)
    at <anonymous>
Origami: Failed to add polyfills
```

Setting up polyfills manually works well, though. 😃👍

Here is my project detail:

```bash
Angular CLI: 6.1.5
Node: 8.9.0
OS: darwin x64
Angular: 6.1.4
... animations, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.7.5
@angular-devkit/build-angular     0.7.5
@angular-devkit/build-optimizer   0.7.5
@angular-devkit/build-webpack     0.7.5
@angular-devkit/core              0.7.5
@angular-devkit/schematics        0.7.5
@angular/cli                      6.1.5
@ngtools/webpack                  6.1.5
@schematics/angular               0.7.5
@schematics/update                0.7.5
rxjs                              6.2.2
typescript                        2.7.2
webpack                           4.9.2
```

**Comment by asyncLiz** [comment]
Probably because your app's tsconfig is missing the `target` property, which is optional. I'll work on a fix!

**Comment by dwiyatci** [comment]
It's actually there under `compilerOptions`:

```json
{
  "compileOnSave": false,
  "compilerOptions": {
    "baseUrl": "./",
    "outDir": "./dist/out-tsc",
    "sourceMap": true,
    "declaration": false,
    "module": "esnext",
    "moduleResolution": "node",
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "target": "es5",
    "typeRoots": [
      "node_modules/@types"
    ],
    "lib": [
      "es2017",
      "dom"
    ]
  }
}
```

**Comment by asyncLiz** [comment]
What does your `angular.json` app's tsconfig point to though? The default is a separate tsconfig file that extends the root. We're not following those extend directives yet, so it doesn't see the target.

**Comment by dwiyatci** [comment]
Right, my `tsconfig.app.json` doesn't have `target` prop.

**Comment by asyncLiz** [comment]
I've got a [todo](https://github.com/hotforfeature/origami/blob/master/bin/lib/polyfill/util.ts#L100-L101) to add that, though I must have missed it in the rush to get everything else. I'll start working on it! 

In the meantime, manually setting up polyfills or adding a target to the `tsconfig.app.json` will bypass the error

**Comment by asyncLiz** [comment]
This should be fixed in 3.0.2 now

**Comment by dwiyatci** [comment]
Gave it a spin just now, works like a charm! :stars: 

Thank you!
